examples/pie_and_polar_charts/bar_of_pie.py

```diff
@@ -0,0 +1,97 @@
+"""
+=========
```

Comment:
This will fail to build because the `=` must be the right length...

Reply:
I was parroting other examples. What is the correct length?

Reply:
As long as the title (or longer I think is OK)